test/faker/datetime_test.exs

```diff
@@ -0,0 +1,44 @@
+if Version.match?(System.version(), ">= 1.3.0") do
+  defmodule DateTimeTest do
+    use ExUnit.Case, async: true
+
+    @microseconds_per_day 86400000000
+
+    test "forward/1" do
+      forwarded_date = Faker.DateTime.forward(10)
+      assert %DateTime{year: year, month: month, day: day, second: second, microsecond: microsecond}
+        = forwarded_date
+      assert now().year > year || now().month > month || now().day > day
+        || now().second > second || now().microsecond > microsecond
+    end
+
+    test "backward/1" do
+      backward_date = Faker.DateTime.backward(10)
+      assert %DateTime{year: year, month: month, day: day, second: second, microsecond: microsecond}
+        = backward_date
+      assert now().year > year || now().month > month || now().day > day
+        || now().second > second || now().microsecond > microsecond
```

Comment:
Are these comparisons against a newly generated `now()` safe? What if it is `2017-12-31 23:59:59.999999` at the start of line 19?